vetiver/handlers/_interface.py

```diff
@@ -1,31 +1,100 @@
-from vetiver.handlers import pytorch_vt, sklearn_vt
-import sklearn 
+from typing import Any
+from vetiver.handlers import pytorch, scikitlearn
+from functools import singledispatch
+import sklearn
 
 torch_exists = True
 try:
     import torch
 except ImportError:
     torch_exists = False
 
-def create_translator(model, ptype_data):
+class InvalidModelError(Exception):
+    """
+    Throw an error if `model` is not
+    from scikit-learn or torch
+    """
+
+    def __init__(
+        self,
+        message="The `model` argument must be a scikit-learn or torch model.",
+    ):
+        self.message = message
+        super().__init__(self.message)
+
+CREATE_PTYPE_TPL =  """\
```

Comment:
WDYT of having this as a doc page? something like...

* VetiverHandler documented here: https://rstudio.github.io/vetiver-python/
* a getting started example of subclassing / using the handler as a separate page (e.g. "Customizing model descriptions and predictions" or something)